cmd/ftl/cmd_dev.go

```diff
@@ -38,6 +41,24 @@ func (d *devCmd) Run(ctx context.Context, client ftlv1connect.ControllerServiceC
 	d.modules = make(map[string]moduleFolderInfo)
 	d.client = client
 
+	stream, err := client.PullSchema(ctx, connect.NewRequest(&ftlv1.PullSchemaRequest{}))
```

Comment:
This should be in the goroutine itself, and should handle disconnects of the stream. ie. there should be _two_ loops, an outer one initiating the stream and an inner one pulling from the stream. Don't forget to close the stream if it errors.